src/events/IonEventStream.ts

```diff
@@ -379,7 +428,25 @@ export class IonEventStream {
         } else {
             this.reader.stepIn();
             for (let tid; tid = this.reader.next();) {
-                annotations.push(this.reader.stringValue()!);
+                if(tid == IonTypes.STRUCT) {
```

Comment:
Here and elsewhere: TS style guides encourage using the "threequals" (`===`) operator by default and only falling back to `==` in situations where that behavior is required.

We can either try to fix this manually in this PR or fix amzn/ion-js#631.

Reply:
I think doing it in  amzn#631 sounds good.